modules/external-api/src/main/java/org/opencastproject/external/endpoint/EventsEndpoint.java

```diff
@@ -697,7 +697,7 @@ private Response scheduleNewEvent(EventHttpServletRequest request, JSONObject sc
           @RestParameter(name = "withscheduling", isRequired = false, description = "Whether the scheduling information should be included in the response.", type = Type.BOOLEAN),
           @RestParameter(name = "withpublications", isRequired = false, description = "Whether the publication ids and urls should be included in the response.", type = Type.BOOLEAN),
           @RestParameter(name = "onlyWithWriteAccess", isRequired = false, description = "Whether only to get the events to which we have write access.", type = Type.BOOLEAN),
-          @RestParameter(name = "filter", isRequired = false, description = "A comma seperated list of filters to limit the results with. A filter is the filter's name followed by a colon \":\" and then the value to filter with so it is the form <Filter Name>:<Value to Filter With>.", type = STRING),
+          @RestParameter(name = "filter", isRequired = false, description = "Usage <Filter Name>:<Value to Filter With>. Filters can combine using a comma \",\". Available Filters: presenters, contributors, location, textFilter, series, subject. If API ver > 1.1.0 also: identifier, title, description, series_name, language, created, license, rightsholder, is_part_of, source, status, agent_id, start, technical_start.", type = STRING),
```

Comment:
> `<Filter Name>:<Value to Filter With>.`

This is interpreted as HTML and thus displays as `:. ` (Firefox 100.0.2). Of course this is also the case in the current 12.x branch and not your fault, but it would be nice if you could fix it while you're at it :)

Reply:
Ohh, I didn't realize that! Thanks for pointing it out!